hikari/api/rest.py

```diff
@@ -4755,6 +4839,202 @@ async def create_guild_category(
             If an internal error occurs on Discord while handling the request.
         """
 
+    @abc.abstractmethod
+    async def create_message_thread(
+        self,
+        channel: snowflakes.SnowflakeishOr[channels_.PermissibleGuildChannel],
+        message: snowflakes.SnowflakeishOr[messages_.PartialMessage],
+        name: str,
+        *,
+        # While there is a "default archive duration" setting this doesn't seem to effect this context
+        # since it always defaults to 1440 minutes if auto_archive_duration is left undefined.
+        auto_archive_duration: undefined.UndefinedOr[time.Intervalish] = datetime.timedelta(minutes=60),
+        reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
+    ) -> typing.Union[channels_.GuildPublicThread, channels_.GuildNewsThread]:
+        """Create a public or news thread on a message in a guild channel.
+
+        !!! warning
+            Public threads can only be made in guild text channels
+            and news threads can only be made in guild news channels.
+
+        Parameters
+        ----------
+        channel : hikari.snowflakes.SnowflakeishOr[hikari.channels.PermissibleGuildChannel]
+            Object or ID of the guild news or text channel to create a public thread in.
+        message : hikari.snowflakes.SnowflakeishOr[hikari.messages.PartialMessage]
+            Object or ID of the message to attach the created thread to.
+        name : str
+            Name of the thread channel.
+
+        Other Parameters
+        ----------------
+        auto_archive_duration : hikari.undefined.UndefinedOr[hikari.internal.time.Intervalish]
```

Comment:
Default not documented

Reply:
i don't know what you mean by that, the default is already in the signature and that shows up in the docs